src/components/products/product-subscriptions/react/ProductSubscriptionsRuntime.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { FluentProvider } from "@fluentui/react-components";
+import { Resolve } from "@paperbits/react/decorators";
+import { Router } from "@paperbits/common/routing";
+import * as Constants from "../../../../constants";
+import { RouteHelper } from "../../../../routing/routeHelper";
+import { UsersService } from "../../../../services";
+import { ProductService } from "../../../../services/productService";
+import { Subscription } from "../../../../models/subscription";
+import { ProductSubscriptionsTable } from "./ProductSubscriptionsTable";
+
+type ProductSubscriptionsProps = {}
+type ProductSubscriptionsFCProps = ProductSubscriptionsProps & {
+    usersService: UsersService;
+    productService: ProductService;
+    productName: string;
+};
+
+const loadSubscriptions = async (usersService: UsersService, productService: ProductService, productName: string) => {
```

Comment:
Return type?

Reply:
it will be inferred correctly, I don't think it's necessary IMHO